terraform/production/variables.tf

```diff
@@ -1,9 +1,9 @@
 variable "environment_name" {
-    type = string
-    default = "production"
+  type    = string
+  default = "production"
 }
 
 variable "project_name" {
-    type = string
-    default = "Housing-Search-Listener"
+  type    = string
+  default = "MTFH"
```

Comment:
Should the project name not be `Housing-Production` to keep it consistent between the other APIs/Listeners?